projects/demo/src/modules/components/calendar/index.html

```diff
@@ -22,79 +22,32 @@
         </p>
 
         <tui-doc-example
-            id="base"
-            heading="Basic"
-            [content]="example1"
-        >
-            <tui-calendar-example-1 />
-        </tui-doc-example>
+            *ngFor="let example of examples; let index = index"
+            [component]="index + 1 | tuiComponent"
+            [content]="index + 1 | tuiExample: 'html,ts'"
```

Comment:
```suggestion
            [content]="index + 1 | tuiExample"
```

not only html, ts, but less also

Reply:
This option to use tuiExample will not display examples where html and ts only. I moved styles to ts files because there was not much of them